app/marketing/tests/tasks/factories/profile_factory.py

```diff
@@ -0,0 +1,12 @@
+import factory
+from dashboard.models import Profile
+
+
+class ProfileFactory(factory.django.DjangoModelFactory):
+    """Create instance of Profile for testing."""
+
+    class Meta:
+        model = Profile
+
+    handle = factory.Sequence(lambda n: "Contributor_%03d" % n)
+    data = {}
```

Comment:
We already have this factory defined. This is part of the `dashboard.tests.factories` We should use that instead of creating a duplicate